xbmc/settings/SettingConditions.cpp

```diff
@@ -249,7 +251,7 @@ bool ProfileLockMode(const std::string& condition,
                      const SettingConstPtr& setting,
                      void* data)
 {
-  char *tmp = NULL;
+  char* tmp = nullptr;
   LockType lock = (LockType)strtol(value.c_str(), &tmp, 0);
```

Comment:
Another style adoption we make now is replacing c casts with c++ casts for type safety.